src/SignalR/docs/specs/TransportProtocols.md

```diff
@@ -186,6 +186,32 @@ Long Polling requires that the client poll the server for new messages. Unlike t
 
 A Poll is established by sending an HTTP GET request to `[endpoint-base]` with the following query string parameters
 
+## HTTP Streaming (HTTP/1, HTTP/2, HTTP/3)
```

Comment:
So are we committing to some of this for 5.0? @anurse

Reply:
We do need to talk about the scope, but putting it in the spec doesn't *technically* require us to implement it, because of transport negotiation. It just means that when we do implement it, this is how it is done :).

I do think this will fit in the 5.0 milestone, but I also think that can be a somewhat orthogonal discussion.

@davidfowl we should include the transport names (as returned by `/negotiate`) in this doc, for this transport and all the others.

Reply:
I won't merge it until we decide.